meta-mender-qemu/recipes-support/libp11/libp11_%.bbappend

```diff
@@ -0,0 +1,4 @@
+DEPENDS = "libtool openssl p11-kit"
+
+SRCREV = "e1210903291b1de9eabcad26e740a4b2fbcca692"
```

Comment:
This is the main point on this PR; you need to convince me why do we need this specific version. I trust your debugging, but it should be justified here in the recipe.

Reply:
the current libp11 in meta-oe has this SRCREV:
```
commit e1210903291b1de9eabcad26e740a4b2fbcca692 (tag: libp11-0.4.9)
Author: Michał Trojnara <Michal.Trojnara@stunnel.org>
Date:   Mon Sep 3 20:08:20 2018 +0200

    released 0.4.9
```
which is a bit old. the one in the present PR is the latest one, after the bump to OpenSSL 1.1.1g:
```
commit c8aca3bb76c1387bfd3ef2f71f97708c45b5175c (HEAD -> master, origin/master,
origin/HEAD)
Author: Jussi Keranen <jussi.keranen@vincit.com>
Date:   Thu Aug 13 10:24:51 2020 +0300

    p11_load: Initialize variable

    The uninitialized variable could be partially undefined if it's
    not written completely in 'C_GetInfo()'

    Signed-off-by: Jussi Keranen <jussike@gmail.com>

commit 71c3baeb10543c2bdaa8e7cb8cdbebf04b32cce5
Author: Pavol Misik <pmisik@gmail.com>
Date:   Fri May 15 20:47:57 2020 +0200

    Bump OpenSSL version to 1.1.1g
```

I thought it would be safer to use libp11 that actually is aligned with 1.1.1.

Reply:
I think this one can be removed, if you think that we should not use the latest libp11 (I think we should).

Reply:
Agreed. And to avoid getting stuck on this version forever, please rename the file to `libp11_0.4.%.bbappend`. This way we will get notified if there's a new version available which doesn't match anymore.